dataplane/src/args.rs

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Open Network Fabric Authors
+
+pub(crate) use clap::Parser;
+#[derive(Parser)]
+#[command(name = "Hedgehog Fabric Gateway dataplane")]
+#[command(version = "1.0")] // FIXME
+#[command(about = "A next-gen dataplane for next-gen fabric gateway", long_about = None)]
+pub(crate) struct CmdArgs {
+    #[arg(long, value_name = "core-id used as main", default_value_t = 2)]
+    main_lcore: u8,
+    #[arg(long, value_name = "map lcore set to cpu set")]
+    lcores: Option<String>,
+    #[arg(long, value_name = "in-memory flag", default_value_t = true)]
+    in_memory: bool,
+    #[arg(long, value_name = "PCI devices to probe")]
+    allow: Vec<String>,
+    #[arg(long, value_name = "huge pages", default_value_t = 8192)]
+    huge_worker_stack: u32,
+    #[arg(long, value_name = "socket memory")]
+    socket_mem: Option<String>,
+    #[arg(long, value_name = "enable/disable telemetry", default_value_t = true)]
```

Comment:
we should remove this argument as I [forcibly disabled DPDK](https://github.com/githedgehog/dpdk-sys/blob/e8a896e986507124b8a33049b6d6137233914a38/nix/dpdk/default.nix#L71) telemetry [in a prior commit to dpdk-sys](https://github.com/githedgehog/dpdk-sys/commit/c35a59b9f59782e6507e09a45e6163a705bac1f1)

(I understand that you are just mirroring the prior, also outdated, parameters here)